From 8a75b77382cd934e726c461ef9803a038993261d Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Fri, 19 Oct 2018 12:13:25 +0200
Subject: [PATCH] Merge android-9.0.0_r12

commit f27ca0931e188a113a0686a538ac3b7c5dc60d5f
Merge: 6c31b79 9a32709
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Sun Jul 8 03:01:28 2018 +0000

    Snap for 4880674 from 9a3270918dbfd0acd961e755bdbac91fc1cd0088 to pi-dr1-release

    Change-Id: Iec34b62e9014e5478ce1da123db9355f4014670f

commit 9a3270918dbfd0acd961e755bdbac91fc1cd0088
Author: Bill Yi <byi@google.com>
Date:   Wed Jul 4 22:13:07 2018 -0700

    Import translations. DO NOT MERGE

    Auto-generated-cl: translation import

    Bug: 64712476
    Change-Id: If33e5b9334d06c454896ffbd018c6529ce40b079

commit 6c31b7983635e4c2a48a9b8ff7599bbd2e5dfc60
Merge: df60589 23e54bf
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Tue Jun 5 03:02:26 2018 +0000

    Snap for 4820872 from 23e54bfae2c770842da1d028ac586552fd303302 to pi-dr1-release

    Change-Id: Ifec923ffac6e4d85694b7b203044de2112b66489

Change-Id: I8375a754c88f2562164652dfc235be1635e5a023
---
 res/values-sw/strings.xml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/res/values-sw/strings.xml b/res/values-sw/strings.xml
index 19b9db8..857f868 100644
--- a/res/values-sw/strings.xml
+++ b/res/values-sw/strings.xml
@@ -6,8 +6,8 @@
     <string name="record_trace" msgid="6416875085186661845">"Rekodi nyayo"</string>
     <string name="trace_debuggable_applications" msgid="7957069895298887899">"Tafuta programu zinazoweza kutatuliwa"</string>
     <string name="categories" msgid="2280163673538611008">"Aina"</string>
-    <string name="restore_default_categories" msgid="5090536794637169521">"Rejesha aina chaguo msingi"</string>
-    <string name="default_categories_restored" msgid="6861683793680564181">"Imerejesha aina chaguo msingi"</string>
+    <string name="restore_default_categories" msgid="5090536794637169521">"Rejesha aina chaguomsingi"</string>
+    <string name="default_categories_restored" msgid="6861683793680564181">"Imerejesha aina chaguomsingi"</string>
     <string name="applications" msgid="521776761270770549">"Maombi"</string>
     <string name="no_debuggable_apps" msgid="4386209254520471208">"Haikupata programu zozote ambazo zinaweza kutatuliwa"</string>
     <string name="buffer_size" msgid="3370134227980578569">"Ukubwa wa hifadhi ya akiba"</string>
-- 
2.17.1

